openquake/hazardlib/gsim/abrahamson_2014.py

```diff
@@ -295,10 +296,12 @@ def _get_soil_depth_term(self, C, z1pt0, vs30):
         """
         # Get reference z1pt0
         z1ref = self._get_z1pt0ref(vs30)
+        # Get z1pt0
+        z10 = copy.deepcopy(z1pt0)
         # This is used for the calculation of the motion on reference rock
         idx = z1pt0 < 0
-        z1pt0[idx] = z1ref[idx]
-        factor = np.log((z1pt0+0.01)/(z1ref+0.01))
+        z10[idx] = z1ref[idx]
+        factor = np.log((z10+0.01)/(z1ref+0.01))
```

Comment:
To be compliant with PEP8 you must put spaces around the mathematical operators. Also, I would have expected a test sensible to this change to be added.